pkg/engine/provider/filesystem_test.go

```diff
@@ -310,7 +315,7 @@ func TestFileSystemSourceProvider_checkConditions(t *testing.T) {
 				path: filepath.FromSlash("assets/queries"),
 			},
 			want: want{
-				got: true,
```

Comment:
I changed this test because I didn't understand why "true" was expected. During the test I passed in this line:
`_, err := os.Stat(filepath.Join(path, "Chart.yaml"))`

Which returned `true, nil `because I had no `Chart.yaml` file in my path, so I understood if there is no `Chart.yaml` file it should be ignored and I don't understand why.

Can you explain me? @cxMiguelSilva

Reply:
Hi @dim-ops, 
That check is in place to ignore all folders of Helm Charts to later make a scan to the Helm templates

Reply:
Yes I understand the interest of `_, err := os.Stat(filepath.Join(path, "Chart.yaml"))`but I don't understand why `true` is returned when Chart.yaml doesn't exist. If I understand well true is when folder must be skipped, if Chart.yaml is not present folder should be parsed.

It's why I modify this check 
```
		_, err := os.Stat(filepath.Join(path, "Chart.yaml"))
		if errors.Is(err, os.ErrNotExist) {
			return false, nil
		} else if err != nil || resolved {
			log.Error().Msgf("failed to check helm: %s", err)
			return true, nil
		}
```